packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4483,13 +4485,26 @@ class Semantics extends SingleChildRenderObjectWidget {
   /// or an explicit [textDirection] should be provided.
   final String label;
 
-  /// The reading direction of the [label].
+  /// Provides a textual description of value of the widget.
```

Comment:
Maybe add some documentation to explain the difference between label and value?
The documentation on SemanticsConfiguration made it clear to me, but I think a user is more likely to interact with Semantics and look for an explanation here...?

Looks like we document the same fields in 3 places: on Semantics, SemanticsConfiguration, and SemanticsData.
Would it be acceptable to keep all documentation here, and just refer to it from the other classes?
If we end up keeping 3 copies of the documentation we should probably make sure the difference between label/value is explained in each of them...